my-webxr-app/src/App.tsx

```diff
@@ -75,19 +111,20 @@ export default function App() {
             <ambientLight />
             <pointLight position={[10, 10, 10]} />
             <Controllers />
-            <Button position={[0, 1.5, -1]} />
-            <RotatingBox position={[0.8, 1.5, -1]} />
-            <RotatingBox position={[-0.8, 1.5, -1]} />
+
             <Axis minValue={minNum} maxValue={maxNum} scaleFactor={scaleFactor} startX={startPointX}
-            startY={startPointY} startZ={startPointZ} endPoint={endPoint} radius={radius}
+            startY={startPointY} startZ={startPointZ} endPoint={Length} radius={radius}
             labelOffset={labelOffset}/>
 
             {/* Temporary display/test of the data points.
               These will eventually be created by the plot itself */}
-            <DataPoint id={0} meshProps={{ position: [0.25, 1.75, -0.75] }} />
-            <DataPoint id={1} meshProps={{ position: [0, 1.75, -0.75] }} />
-            <DataPoint id={2} meshProps={{ position: [-0.25, 1.75, -0.75] }} />
-            <DataPointMenu position={[0, 2.2, -0.75]} />
+            <DataPoint id={0} meshProps={{ position: datapoint1 } } />
```

Comment:
These are the data points from before Aesha’s merge right? Did you forget to merge your branch with ID3 branch before merging to ID3?

Reply:
I merged it with ID3 a couple hours ago, but did not do it just before the merge. But yes this was them before the datapoint changes that aesha made. I believe that her changes would not affect mine, since meshProps{{position:}} is still a thing and thats all my change really touches